Adding Github Actions

mfixstsci opened the pull request:
Removing `.travis.yml` and adding `./github/workflows/ci.yml`.

pep8speaks commented:
Hello @mfixstsci, Thank you for updating !

* In the file [`jwst_coronagraph_visibility/gui.py`](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py):

> [Line 24:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L24): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 25:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L25): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 27:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L27): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 28:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L28): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 29:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L29): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 30:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L30): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 31:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L31): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 32:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L32): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 33:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L33): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 34:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L34): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 44:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L44): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 45:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L45): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 46:1](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L46): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 93:8](https://github.com/spacetelescope/jwst_coronagraph_visibility/blob/31ec1b9209c74551a442cfa881e8cc6d341af463/jwst_coronagraph_visibility/gui.py#L93): [E126](https://duckduckgo.com/?q=pep8%20E126) continuation line over-indented for hanging indent

mfixstsci commented:
@brynickson this is ready for review.

Here is what I've done:

- Removed `.travis.yml`
- Added `.github/workflows/ci.yml` (Github Actions CI)
- Updated the python version requirement in `setup.py`
- Regenerated test data for Fomalhaut in `test_skyvec2ins.py`

To verify that the tests are passing, this see this link to my fork and branch of this repository:
https://github.com/mfixstsci/jwst_coronagraph_visibility/actions
